server.js

```diff
@@ -77,7 +77,8 @@ try {
 const _logger = require('./logger');
 
 const os = require('os');
-const isMobile = os.platform() === 'android' || os.platform() === 'ios' || process.env.FORCE_MOBILE;
+const isMobile = os.platform() === 'android' || process.env.FORCE_MOBILE;
```

Comment:
Would it make sense to rename isMobile to something more accurate, since an iOS device is no longer considered mobile here? I can just foresee myself potentially misusing isMobile in the future.

Reply:
Agreed! I've renamed it and simplified some logic later on in the file that had remnants of the past setup where isMobile had a different meaning

Reply:
Awesome!